README.md

```diff
@@ -104,6 +105,106 @@ public function main() returns error? {
 }
 ```
 
+### Fail-safe CSV parsing with metadata error logging
+
+```ballerina
+import ballerina/data.csv;
+import ballerina/io;
+
+type Transaction record {
+    int transactionId;
+    decimal amount;
+    string date;
+};
+
+public function main() returns error? {
+    string csvString = string `transactionId,amount,date
+                               1001,250.50,2024-01-15
+                               1002,INVALID,2024-01-16
+                               1003,175.25,2024-01-17`;
+
+    Transaction[] transactions = check csv:parseString(csvString, {
+        failSafe: {
+            outputMode: {
+                filePath: "./logs/transaction-errors.log",
+                dataType: csv:METADATA,
+                fileWriteOption: csv:APPEND,
+                enableConsoleLogs: false
+            }
+        }
+    });
+    
+    io:println(transactions);
+}
+```
+
+### Fail-safe CSV parsing with raw data error logging
+
+```ballerina
+import ballerina/data.csv;
+import ballerina/io;
+
+type SensorReading record {
+    int sensorId;
+    float temperature;
+    int timestamp;
+};
+
+public function main() returns error? {
+    string csvString = string `sensorId,temperature,timestamp
+                               1,23.5,1640000000
+                               2,HOT,1640000060
+                               3,22.1,NOT_A_NUMBER
+                               4,24.8,1640000180`;
+
+    SensorReading[] readings = check csv:parseString(csvString, {
+        failSafe: {
+            outputMode: {
+                filePath: "./logs/sensor-errors.log",
+                dataType: csv:RAW,
+                fileWriteOption: csv:OVERWRITE,
+                enableConsoleLogs: false
+            }
+        }
+    });
+    
+    io:println(readings);
+}
```

Comment:
If we add different samples for different data types, we need to show a sample logoutputs for each types. Otherwise its very hard to differentiate

Reply:
Fixed by [1d351fd](https://github.com/ballerina-platform/module-ballerina-data.csv/pull/60/commits/1d351fd5c7b2431cb10fc076b573c271b00cffd7)